src/index.ts

```diff
@@ -53,6 +55,21 @@ class ServerlessPlugin {
 
     // include node_modules into build
     fs.symlinkSync(path.resolve('node_modules'), path.resolve(path.join(buildFolder, 'node_modules')))
+    // include any "extras" from the "include" section
+    if(this.serverless.service.package.include && this.serverless.service.package.include.length > 0){
+      globby(this.serverless.service.package.include).then((files) => {
+        _.forEach(files, (filename) => {
+          let destFileName = path.resolve(path.join(buildFolder, filename));
+          let dirname = path.dirname(destFileName);
+          if(!fs.existsSync(dirname)){
```

Comment:
Please use spaces like so: `if (!fs.existsSync(dirname)) {`

(Also applies to other places)